scripts/list_needed_libs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+set -e
+
+# this file lists all dynamic dependenices of executables in gprofiler/resources.
+# we use it to let staticx know which libraries it should pack inside.
+# (staticx knows to pack the libraries used by the executable we're packing. it doesn't know
+# which executables are to be used by it)
+
+BINS=$(find gprofiler/resources -executable -type f)
+
+libs=
+
+for f in $BINS ; do
+    set +e
+    ldd_output=$(ldd $f 2>&1)
```

Comment:
```suggestion
    IFS= ldd_output="$(ldd $f 2>&1)"
```

Reply:
Why is the `IFS=` needed? To avoid any mashup if the output contains `"`?

Reply:
Not needed apparently because bash preserves the value when it's quoted.

Reply:
kk